tests/contrib/data/url2purl.json

```diff
@@ -139,5 +139,33 @@
 
   "https://crates.io/api/v1/crates/rand/0.7.2/download": "pkg:cargo/rand@0.7.2",
   "https://crates.io/api/v1/crates/clap/2.33.0/download" : "pkg:cargo/clap@2.33.0",
-  "https://crates.io/api/v1/crates/structopt/0.3.11/download" : "pkg:cargo/structopt@0.3.11"
+  "https://crates.io/api/v1/crates/structopt/0.3.11/download" : "pkg:cargo/structopt@0.3.11",
```

Comment:
please use consistently no space before and one space after the colon (including fixing other entries above as needed.